src/tools/rest-backend/.gitignore

```diff
@@ -0,0 +1,3 @@
+CMakeCache.txt
+nbproject/
```

Comment:
CMakeCache.txt appeared in the source for some reason, maybe netbeans tries to compile something, I don't really know. I added comments to the gitignore.